include/swift/AST/Decl.h

```diff
@@ -5812,6 +5812,9 @@ class FuncDecl : public AbstractFunctionDecl {
   bool isConsuming() const {
     return getSelfAccessKind() == SelfAccessKind::__Consuming;
   }
+  bool isCallable() const {
+    return getName().str() == "call" && isInstanceMember();
```

Comment:
Consider adding an `IsCallable` bit to `FuncDecl`?

@lattner's [response](https://github.com/apple/swift/pull/24122#discussion_r277122262):

> I'd recommend _not_ doing that, unless this is a particularly hot predicate. Cached bits like this have to be kept in sync with the ast. Also, instead of serializing it, if the bit needs to be stored, it can always be computed in the module reader.